go/vt/vtgate/planbuilder/plan_test.go

```diff
@@ -617,6 +618,45 @@ func BenchmarkPlanner(b *testing.B) {
 	}
 }
 
+func BenchmarkSelectVsDML(b *testing.B) {
+	vschema := &vschemaWrapper{
+		v:             loadSchema(b, "schema_test.json"),
+		sysVarEnabled: true,
+		version:       V4,
+	}
+
+	var dmlCases []testCase
+	var selectCases []testCase
+
+	for tc := range iterateExecFile("dml_cases.txt") {
+		if tc.output2ndPlanner != "" {
```

Comment:
`output2ndPlanner` is an artifact of this project: #7280

it's the plan generated by the new gen4 planner. for this work, you should just ignore it and add all queries.